### ISSUE TITLE
Propcheck gives a warning when run on elixir 1.17

### ISSUE THREAD

**Issue by zkessin** [alfert/propcheck#229]
I am running propcheck version 1.4 with elixir 1.17 and am getting this warning 

```
    warning: ExUnit.Case.register_test/4 is deprecated. Use register_test/6 instead
    │
  5 │   property "my property", [], _context do
    │   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    │

```

**Comment by brettwise** [comment]
Fix has been merged but I guess not released: https://github.com/alfert/propcheck/pull/227

**Comment by brettwise** [comment]
@zkessin until they cut a new release with it one solution is to specify the SHA that contains the updated code by pointing your dep at it directly like:

```
      {:propcheck, github: "alfert/propcheck", ref: "c564e89d3873caf9c6bf64a2af4bb3890e24ecf1", only: [:test, :dev]},
```

**Comment by edennis** [comment]
Thanks for this awesome library! Any chance at cutting a new release? 🙏🏻

**Comment by evnu** [comment]
Hopefully we can release a new version of PropCheck soon, see https://github.com/alfert/propcheck/pull/230 for a bump of PropEr which blocked releasing a new version of PropCheck before.

**Comment by evnu** [comment]
The fix has been released with [v1.5.0](https://hex.pm/packages/propcheck/1.5.0)
